x-pack/plugins/observability_solution/infra/server/lib/alerting/inventory_metric_threshold/lib/convert_metric_value.ts

```diff
@@ -18,4 +18,6 @@ export const convertMetricValue = (metric: SnapshotMetricType, value: number) =>
 const converters: Record<string, (n: number) => number> = {
   cpu: (n) => Number(n) / 100,
   memory: (n) => Number(n) / 100,
+  tx: (n) => Number(n) / 8,
```

Comment:
I didn't find an easy way to add a test for this logic, any suggestion is welcome!

Reply:
How about just adding a Jest unit test?

Reply:
Added some unit tests at a lower level than the executor.